standard_logger.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+
+def login():
```

Comment:
Can you explain in more detail what's the purpose here?
We could log with `logging.info` directly. Do you want to create a wrapper module for the logging module?